tests/test_modeling_marian.py

```diff
@@ -44,6 +65,23 @@ def model(self):
         else:
             return model
 
+    def _test_repl_generate_batch(self):
+        model_inputs: dict = self.tokenizer.prepare_translation_batch(src_texts=self.src_text).to(torch_device)
+        self.assertEqual(self.model.device, model_inputs["input_ids"].device)
+        generated_ids = self.model.generate(
+            model_inputs["input_ids"],
+            attention_mask=model_inputs["attention_mask"],
+            length_penalty=1.0,  # same as C++
+            num_beams=2,  # 6 is the default
+            # no_repeat_ngram_size=1,
+            # bad_words_ids=[[self.tokenizer.pad_token_id]],
+            decoder_start_token_id=self.tokenizer.pad_token_id,  # mimics 0 embedding at first step.
```

Comment:
This is always the case no? Maybe set `decoder_start_token_id = pad_token_id` in `MarianConfig`

Reply:
done!